app/pages/about/team-page.cjsx

```diff
@@ -176,6 +176,9 @@ counterpart.registerTranslations 'en',
         bio: '''Associate Professor of Physics and Astronomy at the University of Minnesota. One of the Zooniverse
         co-founders, she helps with project wrangling (aka management) and strategic planning while trying
         to squeeze a bit of galaxy evolution science in.'''
+      mariamSaeedi:
+        title: 'Web Developer Intern'
+        bio: '''To do'''
```

Comment:
I'd add in an actual bio, rather than show 'To Do' on a production site. Can just be something like 'Mariam is the 2017 Zooniverse intern.` for now.